core/validation/failures.py

```diff
@@ -248,7 +253,7 @@ def to_user_centered_components(self) -> tuple[str, str, str]:
 
 
 @dataclass
-class PreTransformationFailure(ValidationFailure):
+class PreTransformationFailure(PreTransFormationFailureClass, TFUCFailureMessage):
```

Comment:
Can this be renamed to `WrongInputFailure` or something? And then we can just name the base class `PreTransformationFailure`

Reply:
Renamed